polars/polars-core/src/chunked_array/ops/unique/rank.rs

```diff
@@ -1,12 +1,16 @@
 use crate::prelude::*;
 
+use rand::prelude::SliceRandom;
```

Comment:
If we conditionally compile, these can be imported in the branch.

Reply:
So where should it be added?

Reply:
This is also fine. :+1: